run-full-regression-test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+##
+##  Utility script for running regression tests on a developer workstation.
+##  It is an automation of the procedure described in the docs/TEST.md file.
+##  It is a very good idea to run this script frequently, and at least before
+##  committing something in a PR.
+##
+
+
+DEPS="gradle openssl"
+for dep in DEPS ; do
```

Comment:
This is a bug, it's fixed in a branch that's already being prepared.  It's minor so I'll let that pass for now.